test-alerts.cpp

```diff
@@ -5,4 +5,22 @@
 
 TEST_CASE("infers the breach according to limits") {
   REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
+  /* InferBreach testcases */
+  REQUIRE(inferBreach(35, 20, 30) == TOO_HIGH);
+  REQUIRE(inferBreach(20, 20, 20) == NORMAL);
+  /* classifyTemperatureBreach testcases */
+  REQUIRE(classifyTemperatureBreach(PASSIVE_COOLING, 12) == NORMAL);
+  REQUIRE(classifyTemperatureBreach(HI_ACTIVE_COOLING, 12) == NORMAL);
+  REQUIRE(classifyTemperatureBreach(MED_ACTIVE_COOLING, 12) == NORMAL);
+  
+  REQUIRE(classifyTemperatureBreach(PASSIVE_COOLING, -1) == TOO_LOW);
+  REQUIRE(classifyTemperatureBreach(HI_ACTIVE_COOLING, 41) == TOO_HIGH);
```

Comment:
Missing coverage tests for alert targets...
Coverage can be improved...

Reply:
Written new test cases and improved coverage.

Reply:
👍